web/packages/teleport/src/Discover/ConnectMyComputer/SetupConnect/SetupConnect.tsx

```diff
@@ -39,6 +62,89 @@ export function SetupConnect(props: AgentStepProps) {
     path: Path.ConnectMyComputer,
   });
 
+  const { node, isPolling } = usePollForConnectMyComputerNode({
+    username,
+    clusterId,
+    pingInterval,
+  });
+
+  const [showHint, setShowHint] = useState(false);
+  useEffect(() => {
+    if (isPolling) {
+      const id = window.setTimeout(() => setShowHint(true), showHintTimeout);
+
+      return () => window.clearTimeout(id);
+    }
+  }, [isPolling, showHintTimeout]);
+
+  let pollingStatus: JSX.Element;
+  if (showHint && !node) {
+    pollingStatus = (
+      // Override max-width to match StyledBox's max-width.
+      <HintBox header="We're still looking for your computer" maxWidth="800px">
+        <Flex flexDirection="column" gap={3}>
+          <Text>
+            There are a couple of possible reasons for why we haven't been able
+            to detect your computer.
+          </Text>
+
+          <ul
+            css={`
+              margin: 0;
+              padding-left: ${p => p.theme.space[3]}px;
+            `}
+          >
+            <li>
+              <Text>
+                You did not start Connect My Computer in Teleport Connect yet.
+              </Text>
+            </li>
+            <li>
+              <Text>
+                The Teleport agent started by Teleport Connect could not join
+                this Teleport cluster. Check if the Connect My Computer tab in
+                Teleport Connect shows any error messages.
+              </Text>
+            </li>
+            <li>
+              <Text>
+                The computer you are trying to add has already joined the
+                Teleport cluster before you entered this page.
+              </Text>
```

Comment:
> if this is the case, can the user just exit this page? like setup is `complete`? (i'm asking because this text feels incomplete, like "tell me what to do")

Good point. Yeah, they can exit the page. I suppose I could ask something like "If that's the case, go back to the resources to connect to it." and maybe even link to the resources page. I don't know how we usually refer to the resources page in the UI hence just "the resources" in my proposed message.

> coming in blind, i got confused what `setting up teleport connect` had to do with `pinging`. i'm sorry, i should've questioned it from your first PR.

Does that mean there are things I should further clarify? 😶‍🌫️ Or did I not explain the purpose behind the flow clearly enough but after reading the code you get how pinging is related to this step? Or is the feedback completely from the perspective of a user?

> I think the flow is doing this right? -> `It can also connect your computer as an SSH resource` (which was confusing because it sounded like an optional thing we can do)

Yes, the flow is for connecting your computer to the cluster as an SSH resource by using Teleport Connect to do this. I see how it can sound like something optional. I guess what we tried to capture here is a general description of what Connect is for and that it's not only for adding your computer to the cluster, or perhaps that it's not the main feature of the app, but we're going to use it to connect the computer to the cluster. I can talk with Kenny about this.

> I think we should clarify the title of this flow to be something like `Connect your computer as an SSH resource using Teleport Connect`

By the title of this flow you mean the current `<Header>Set Up Teleport Connect</Header>` or the title of this step in the flow?

Reply:
> Or is the feedback completely from the perspective of a user?

This. 

> By the title of this flow you mean the current <Header>Set Up Teleport Connect</Header> or the title of this step in the flow?

The header. B/c it's so much bigger it grabs my attention and i may or may not have read the other smaller texts lol, so it just sounded like `download connect and run it` but we are doing more than that. 

idk maybe it's just me and how weirdly i interpret things 🤷‍♀️